Examples/Tests/Larmor/inputs.mr

```diff
@@ -2,7 +2,7 @@
 max_step = 400
 
 # number of grid points
-amr.n_cell =  32  32
+amr.n_cell =  64  64
```

Comment:
This was required because the number of cells is otherwise too small with 2 MPI procs. (WarpX complains about in the PML code.)